dxr/plugins/rust/refs.py

```diff
@@ -126,6 +126,27 @@ def menu_items(self):
         return menu
 
 
+class MacroRef(_RustRef):
+    def prepare_menu_data(self, tree_index, datum):
+        self.hover = datum['qualname']
+        return trim_dict(datum, ['qualname'])
+
+    def menu_items(self):
```

Comment:
Oh yeah, sorry, should have looked that up, it wasn't what I was expecting. Anyway, yeah, I guess you should add `find_references_...` to RefRef